assets/src/edit-story/components/keyboardShortcutsMenu/karma/keyboardShortcuts.karma.js

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { within } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../karma';
+
+describe('Keyboard Shortcuts Menu', () => {
+  let fixture;
+
+  beforeEach(async () => {
+    fixture = new Fixture();
+    fixture.setFlags({ showKeyboardShortcutsButton: true });
+
+    await fixture.render();
+  });
+
+  afterEach(() => {
+    fixture.restore();
+  });
+
+  describe('CUJ: User can interact with menu using mouse: Click toggle button to open, click close button to close menu', () => {
+    it('should be able to open menu by clicking on keyboard shortcut button', async () => {
+      const hiddenMenu = fixture.screen.queryByRole('dialog', {
+        name: /^Keyboard Shortcuts Menu$/,
```

Comment:
nit: since you're grabbing this button multiple times it might be nice to save the regex value as a variable so that if the text ever updates you only have one spot to update it here.

Reply:
Ooo, I like that, I'll constant-ify them!

Reply:
Added selector functions so the whole statement isn't repeated a bunch of times now 🎉